src/fold/composites/target.py

```diff
@@ -84,11 +84,6 @@ def preprocess_primary(
         else:
             return y.to_frame(), None
 
-    def postprocess_result_primary(
```

Comment:
this is now the default implementation, so there's no need to override it just to throw an exception!